package.json

```diff
@@ -63,7 +63,7 @@
     "coverage:test:package": "nyc node_modules/mocha/bin/_mocha --opts mocha.opts \"gh-badges/**/*.spec.js\"",
     "coverage:test:integration": "nyc node_modules/mocha/bin/_mocha --opts mocha.opts \"lib/**/*.integration.js\" \"services/**/*.integration.js\"",
     "coverage:test:services": "nyc node_modules/mocha/bin/_mocha --opts mocha.opts --delay lib/service-test-runner/cli.js",
-    "coverage:test": "rimraf .nyc_output coverage; npm run coverage:test:server; npm run coverage:test:package; npm run coverage:test:frontend; npm run coverage:test:integration; npm run coverage:test:services",
+    "coverage:test": "rimraf .nyc_output coverage; npm run defs; npm run coverage:test:server; npm run coverage:test:package; npm run coverage:test:frontend; npm run coverage:test:integration; npm run coverage:test:services",
```

Comment:
will `npm run defs` be needed on the `test:js:frontend` script too or is that something coverage specific? 

Probably out of scope here but any thoughts/opinions on utilizing the `pre`/`post` npm scripts (like `precoverage:test`) as well as chaining scripts together to avoid some of the repetition? 

For example if we did something like:
```json
  "coverage:test:frontend": "BABEL_ENV=test nyc --nycrc-path .nycrc-frontend.json npm run test:js:frontend"
```

Reply:
Yea, it is needed. It does get run by `test`. I'm open to revisiting that, though there's a tradeoff in running it which is that running extra stuff slows down the tests between iterations.

I realized I forgot `features`, which is the task we actually need here, along with `defs`. 😝 

Does that chaining work with `nyc`? If it does, that would be _excellent_. Maintaining these test invocations can be a bit exhausting.

Reply:
> Does that chaining work with `nyc`? If it does, that would be _excellent_. Maintaining these test invocations can be a bit exhausting.

It's always worked for me (example [here](https://github.com/swellaby/vsts-bump/blob/master/package.json#L25)) so it _should_ work here too (knock on wood!)

Reply:
Ah, nice. I like how you've moved things into multiple mocha.opts files. We should look at that after they finish overhauling their config.